### PR TITLE
Fix ext storage test class name in test script

### PR DESCRIPTION
**Pull request by PVince81**
@nickvergessen @rullzer another one, please review

Needed to be able to run unit tests for ext storage backends

### CODE REVIEW

File: autotest-external.sh [hunk 1]
```diff
@@ -198,7 +198,9 @@ EOF
 			# getting backend to test from filename
 			# it's the part between the dots startSomething.TestToRun.sh
 			testToRun=`echo $startFile | cut -d '-' -f 2`
-			testToRun="${testToRun}test.php"
+			# capitalize first letter
```
Comment (nickvergessen) [inline autotest-external.sh]: Is that enough, with things like SFTPTest.php and so on?

Comment (PVince81) [inline autotest-external.sh]: I'll rename the classes to match, unless you want to have fun with changing the shell script to find the correct name or have it configurable in some other place ?

Comment (nickvergessen) [inline autotest-external.sh]: :see_no_evil: No

Comment (nickvergessen) [inline autotest-external.sh]: No means "go ahead and rename"

Comment (nickvergessen) [inline autotest-external.sh]: pushed

**Comment by PVince81** [comment]
:+1: for the renames

**Comment by nickvergessen** [comment]
:+1: for the other part

**Comment by PVince81** [comment]
Ah damn... So many missed merges :frowning:

**Pull request merged**